swagger.yaml

```diff
@@ -4404,7 +8680,7 @@ definitions:
   connectivity-remote-host:
     type: object
     properties:
-      host_id:
+      agent_id:
```

Comment:
for v1 it should remain host_id

Reply:
I guess that we should have a separate definition for 'connectivity-remote-agent' with 'agent_id' in it

Reply:
Already have connectivity-remote-agent

Reply:
right, but this definition is 'connectivity-remote-host' and here we should have 'host_id'
at 'connectivity-remote-agent' I expect to see 'agent_id'

Reply:
Yes already fixed that, thanks